py/modbuiltins.c

```diff
@@ -378,7 +378,16 @@ MP_DEFINE_CONST_FUN_OBJ_1(mp_builtin_ord_obj, mp_builtin_ord);
 STATIC mp_obj_t mp_builtin_pow(size_t n_args, const mp_obj_t *args) {
     switch (n_args) {
         case 2: return mp_binary_op(MP_BINARY_OP_POWER, args[0], args[1]);
-        default: return mp_binary_op(MP_BINARY_OP_MODULO, mp_binary_op(MP_BINARY_OP_POWER, args[0], args[1]), args[2]); // TODO optimise...
+        default:
+#if !MICROPY_INT_POW3
+            mp_raise_msg(&mp_type_NotImplementedError, "3-arg pow() not supported");
+#else
```

Comment:
OK. This is a style thing I guess. I use tend to use #elif only when the first and second conditionals are testing for the same sort of condition (e.g. is it pink else is it green else is it blue?) and use nested ifs when they test different things (e.g. is it pink else (is it square else is it round?)?). The outer test is testing if POW3 is enabled at all while the inner test is for what sort of implementation to use if it is enabled. I'm happy to change it but I personally find this structure shows the intent more clearly.